src/EnergyPlus/HVACManager.cc

```diff
@@ -2394,22 +2267,22 @@ void ReportInfiltrations(EnergyPlusData &state)
                 thisInfiltration.MCpI_temp * (thisZoneHB.MAT - thisZone.OutDryBulbTemp) * TimeStepSysSec * ADSCorrectionFactor;
             thisInfiltration.InfilHeatGain = 0.0;
 
-        } else if (thisZoneHB.MAT <= thisZone.OutDryBulbTemp) {
+        } else {
```

Comment:
[src/EnergyPlus/HVACManager.cc:2397]:(style),[multiCondition],Expression is always true because 'else if' condition is opposite to previous condition at line 2391.